src/components/shared/TooltipHeaderWithDuration.js

```diff
@@ -0,0 +1,39 @@
+// @flow
+
+import React, { PureComponent } from 'react';
+import classNames from 'classnames';
+
+type Props = {
+  duration: number,
+  title: string,
+  className?: string,
+};
+
+export default class TooltipHeaderWithDuration extends PureComponent {
```

Comment:
Could we make this less abstract and just call it `MarkerTooltipContents` or something? Then we could encapsulate some of the display logic that scattered around a bit.